go/stellar/accountmerge.go

```diff
@@ -0,0 +1,40 @@
+package stellar
+
+import (
+	"fmt"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/stellar1"
+	"github.com/keybase/client/go/stellar/stellarcommon"
+	"github.com/keybase/stellarnet"
+)
+
+func AccountMerge(mctx libkb.MetaContext, walletState *WalletState, arg stellar1.AccountMergeCLILocalArg) (res stellarnet.SignResult, from stellar1.AccountID, err error) {
+	baseFee := walletState.BaseFee(mctx)
+	sp, unlock := NewClaimSeqnoProvider(mctx, walletState)
+	defer unlock()
+	tb, err := getTimeboundsForSending(mctx, walletState)
+	if err != nil {
+		return res, from, err
+	}
+
+	bundleEntry, accountBundle, err := LookupSender(mctx, arg.From)
+	if err != nil {
+		return res, from, fmt.Errorf("account merge error looking up <from>: %v", err)
+	}
+	fromSeed := stellarnet.SeedStr(accountBundle.Signers[0].SecureNoLogString())
```

Comment:
Check Signers array length. Might it be zero for mobile-only accounts? Even if not it'd be nice future proofing

Reply:
agreed